coderedcms/importexport.py

```diff
@@ -36,11 +36,11 @@ def import_pages(import_data, parent_page):
     page_content_type = ContentType.objects.get_for_model(Page)
 
     for page_record in import_data['pages']:
-        # build a base Page instance from the exported content (so that we pick up its title and other
+        # build a base Page instance from the exported content (so that we pick up its title and other  # noqa
```

Comment:
Comments can and should be broken up to fit within the character limit. Just add another comment line below to handle the line break / wrap.